example/provider-extensions/seed/configure-seed.sh

```diff
@@ -104,6 +104,7 @@ ensure-config-file "$REPO_ROOT_DIR"/example/provider-extensions/garden/controlpl
 ensure-config-file "$REPO_ROOT_DIR"/example/provider-extensions/garden/project/credentials/infrastructure-secrets.yaml
 ensure-config-file "$REPO_ROOT_DIR"/example/provider-extensions/garden/project/credentials/secretbindings.yaml
 ensure-config-file "$REPO_ROOT_DIR"/example/provider-extensions/"$gardenlet_values" "$REPO_ROOT_DIR"/example/provider-extensions/gardenlet/values.yaml.tmpl
+ensure-config-file "$REPO_ROOT_DIR"/example/provider-extensions/garden/project/project.yaml
```

Comment:
There is downside with the new templates and the `ensure-config-file` function. `infrastructure-secrets.yaml.tmpl`, `secretbindings.yaml.tmpl` and `project.yaml.tmpl` contain invalid data.
Thus, when you start from a fresh installation, those files are copied without any error. Then, the preparation of the seed cluster starts, skaffold builds the Gardener controlplane components and finally there will be an error when skaffold deploys the yaml files.
I would prefer to check for the existence of the yaml files with `check-not-initial` here. This leaves it to the user to copy those files and enter the missing values. In this case `make gardener-extensions-up` fails immediately and not after a couple of minutes.
Alternatively, you you `ensure-config-file` first and then `check-not-initial` for some attributes of these files similar to how it is done for `domain-secrets.yaml`.

Reply:
I think I get what you are talking about, especially after I had several consecutive failures of `make gardener-extensions-up` the first time I tried it . All the time it was complaining about some broken file that was not reported in the previous attempt.  

I will take a look on this.

Reply:
I pushed some changes with 4a3610f5cb284bc01cab9a1081da359551acd705 that should address your concerns, can you please take a look again?

Reply:
yes, looks good now. Thanks 😄